projects/ng-components-docs/src/app/examples/components/radio-group/examples/radio-groups/radio-groups.component.html

```diff
@@ -0,0 +1,25 @@
+<!--
+title: Radio Group
+componentid: component-radio-group
+variantid: component-radio-group-all
+url: https://sebgroup.github.io/ng-components/components/radio-group/examples
+-->
+
+<div class="row">
+    <div class="col">
+        <sebng-radio-group [id]="'simple'" [label]="'Simple'" [list]="list"></sebng-radio-group>
```

Comment:
`Simple` here is a string right ?If so, Is there any reason why you are binding it to label instead of just passing it the usual `label="Simple"` ?

Reply:
I've experienced issues with `prop="string"` on Angular 9. I try to do `[props]="'string'"` instead now